### DISCUSSION TITLE
Releases page empty for anyone else?

### DISCUSSION THREAD

**Discussion by jdork0** [valkey-io/valkey#903] [General]
I'm just getting an empty pages here:

https://github.com/valkey-io/valkey/releases
https://github.com/valkey-io/valkey/tags

**Comment by madolson** [comment]
The releases still exist, it seems like just the tags got reverted somehow: https://github.com/valkey-io/valkey/releases/tag/7.2.6.

**Reply by madolson** [reply]
Ok, I found a backup with all the tags, so they were all pushed again. I'm adding tag protection right now, most likely it seems like someone did a `git tag push` without any tags?

**Comment by jdork0** [comment]
Thanks @madolson!
